test/integration_test.rb

```diff
@@ -44,6 +44,27 @@ def test_tests_are_selected_based_on_changes_in_current_working_tree
         output = rake('ttnt:test:run')
         assert_match '2 runs, 2 assertions, 2 failures', output[:stdout]
       end
+
+      def test_isolated
+        # Make TTNT select all tests
+        git_rm_and_commit("#{@repo.workdir}/.ttnt", 'Remove .ttnt')
+        ENV['ISOLATED'] = '1'
+        output = rake('ttnt:test:run')
+        ENV.delete('ISOLATED')
```

Comment:
You should rather put that under an `ensure` statement like:

~~~ruby
def test_isolated
  # ...
ensure
  ENV.delete('ISOLATED')
end
~~~